src/Events/Services/OutboundService.cs

```diff
@@ -138,6 +146,16 @@ private static CloudEventEnvelope MapToEnvelope(CloudEvent cloudEvent, Subscript
             return cloudEventEnvelope;
         }
 
+        private static string GetAuthorizationCacheKey(string resource, string consumer)
+        {
+            if (resource == null)
```

Comment:
Is there a chance for resource to be null? Would it even work with a cache key of null?

Reply:
Maybe we could rename to setResourceIfNotDefined to be more precise?

Reply:
Not a real possibility, We ensure all events have a resource before they are placed in inbound queue. I'll remove the check.